src/lambda_codebase/initial_commit/bootstrap_repository/adf-build/shared/helpers/terraform/get_accounts.py

```diff
@@ -0,0 +1,128 @@
+import boto3
+import json
+import os
+from paginator import paginator
+
+
+sts = boto3.client('sts')
+
+master_acc_id = os.environ["MASTER_ACCOUNT_ID"]
+if("TARGET_OUS" in os.environ):
+    ou_path = os.environ["TARGET_OUS"]
```

Comment:
Could you also make this UPPER case, to highlight that we are talking about a global here?